cleanlab/object_detection/__init__.py

```diff
@@ -0,0 +1 @@
+from . import rank
```

Comment:
```suggestion
from . import rank
from . import filter
```
Maybe this is required?